host-configs/lassen-blueos_3_ppc64le_ib_p9-clang@10.0.1.cmake

```diff
@@ -69,29 +71,34 @@ set(BLT_MPI_COMMAND_APPEND "mpibind" CACHE STRING "")
 # Cuda
 #------------------------------------------------
 
-set(CUDAToolkit_ROOT "/usr/tce/packages/cuda/cuda-11.2.0" CACHE PATH "")
+set(CUDAToolkit_ROOT "/usr/tce/packages/cuda/cuda-12.0.0" CACHE PATH "")
```

Comment:
updating to CUDA 12 is now able to happen after Spack fixed a bug i reported. We should update to CUDA 12 in a separate PR.